File: app/src/main/java/com/ericneidhardt/dynamicsoundboard/misc/Logger.java
package com.ericneidhardt.dynamicsoundboard.misc;

import android.util.Log;

/**
 * Created by eric.neidhardt on 27.08.2014.
 */
public class Logger {

	private static final boolean ENABLE_LOGGING = true;

	public static void e(String TAG, String msg)
	{
		if (ENABLE_LOGGING)
			Log.e(TAG, msg);
	}

	public static void d(String TAG, String msg)
	{
		if (ENABLE_LOGGING)
			Log.d(TAG, msg);
	}

}

File: app/src/main/kotlin/org/neidhardt/dynamicsoundboard/logger/Logger.kt
package org.neidhardt.dynamicsoundboard.logger

import android.util.Log

/**
 * Project created by eric.neidhardt on 27.08.2014.
 */
object Logger {

	fun e(TAG: String, msg: String?) {
		Log.e(TAG, msg)
	}

	fun d(TAG: String, msg: String?) {
		Log.d(TAG, msg)
    }
}
